DOC-1165/add-docs-about-cancelling-sdds-and-checks-in-the-merchant-section (#367)
swan-io/docs: docs/topics/merchants/online/checks/index.mdx

To [query this limit](./guide-request.mdx), use the `paymentAmountLimit` field at the `merchantPaymentMethod` level in the API.

## Canceling payments {#canceling}

Merchants can cancel check payments using the [`cancelMerchantPayment`](https://api-reference.swan.io/mutations/cancel-merchant-payment/) mutation.

> To understand if a payment is cancelable, check the `availableToCancel` balance of the payment.

Reviewer: ```suggestion
To understand if a payment is cancelable, check its `availableToCancel` balance.
```

Only the full amount can be canceled.

Checks can be canceled up until the point when they're received by the check processing center.
The exact timing depends on postal services, so cancellation availability varies.

> Once the check processing center receives the check, it cannot be canceled.

Reviewer: ```suggestion
Once the check processing center receives the check, it can't be canceled.
```
